bokehjs/gulp/tasks/scripts.coffee

```diff
@@ -62,6 +62,7 @@ gulp.task "scripts:tsjs", ["scripts:coffee", "scripts:js", "scripts:ts"], () ->
         gutil.log("#{gutil.colors.red(real)}#{rest}")
         return
 
+      # XXX: can't enable "6133", because CS generates faulty code for closures
```

Comment:
Bummer. Can we just (not now) replace all of these with a `range(0, 10)` that we write?